draft-ietf-tls-tls13.md

```diff
@@ -951,7 +952,7 @@ message MAY be fragmented across several records).
 
        enum {
            reserved(20), alert(21), handshake(22),
-           application_data(23), (255)
+           application_data(23), update(24), (255)
```

Comment:
I'm going to argue for this being a handshake message later, but this is fine for now.